packages/rocketchat-slashcommands-leave/leave.js

```diff
@@ -0,0 +1,31 @@
+
+/*
+ * Leave is a named function that will replace /leave commands
+ * @param {Object} message - The message object
+ */
+	if (Meteor.isClient) {
+		RocketChat.slashCommands.add('leave', null, {
+			description: 'Leave_the_current_channel'
+		});
+		RocketChat.slashCommands.add('part', null, {
+			description: 'Leave_the_current_channel'
+		});
+	} else {
+		const Leave = function Leave(command, params, item) {
+			if (command !== 'leave' || command !== 'part') {
```

Comment:
If you invert a condition you need to invert the conditionals too:
```
if(command == "leave" || command == "part")
  continue
```
Should be:
```
if (command !== 'leave' && command !== 'part') {
  return
```

In your conde command `part` will execute the `return` cuz it's different from `leave`